config/Config-build.in

```diff
@@ -328,4 +328,14 @@ menu "Global build settings"
 			bool "Full"
 	endchoice
 
+	config CONFIG_TARGET_ROOTFS_SECURITY_LABELS
```

Comment:
This then results in `CONFIG_CONFIG_TARGET_ROOTFS_SECURITY_LABELS`. Please remove the redundant `CONFIG_` prefix.

Reply:
Fixed with latest push. Sorry, my test build failed about the same time you reported this.